src/app/conf/2025/page.tsx

```diff
@@ -22,23 +22,23 @@ export default function Page() {
   return (
     <main className="gql-all-anchors-focusable antialiased">
       <Hero />
-      <div className="gql-conf-container text-neu-900">
+      <div className="gql-conf-container gql-conf-navbar-strip before:dark:bg-blk/30 text-neu-900 before:bg-white/40">
```

Comment:
@saihaj lmk if you think this should have a syntax like `navbar-strip:bg-white/40 dark:navbar-strip:bg-blk/40` powered by `matchUtilities` (probably not worth it? I'm not sure)

Reply:
I think this is fine. Eventually when we migrate the full site this would get cleaned up